modules/spo-selinux-runasany.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+:_content-type: CONCEPT
+[id="spo-selinux-runasany_{context}"]
+
+= About `seLinuxContext: RunAsAny`
```

Comment:
Rm backticks from headings per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) (also it's making the "About" be bold in the TOC somehow 🤔)